src/components/container/InboxItemList/InboxItemList.css

```diff
@@ -0,0 +1,8 @@
+.mdhui-inbox-item-list-title {
+    color: var(--mdhui-text-color-2);
+}
+
+.mdhui-inbox-item-list-empty-text {
+    color: var(--mdhui-text-color-3);
+    padding: 16px 0;
```

Comment:
var --mdhui-padding-md

Reply:
I wasn't aware of those variables, but I'm also a little hesitant to use them.

I think sharing colors (and maybe some other things) via variables makes sense, but I'm concerned that sharing dimensional settings could mean we end up with unforeseen or unintended consequences if/when we ever adjust those.  For example, if we decide at some point that `--mdhui-padding-md` should be `12px` or `20px`, we'll need to go through and find all the components that are using that variable and make sure things still rendered correctly.

I'd be interested to hear others thoughts on it though.  I'm not totally opposed to it, but it just doesn't feel as obvious to me to be something we should use vars for, like colors.

Reply:
Having the variable is good for consistency, especially for across container/presentation components. For views, I could see more wiggle room on not using.
If we do not want to use them at all, is there value in keeping the variable?

Reply:
I guess I'm still not quite sold on it, at least not at this general of a level.

If the variable were something like `--mdhui-padding-for-empty-list-text`, then it would make more sense to me as something we are using to have a consistent look and feel for empty text across our list components.  However, just having the padding set to a generic "medium" padding seems less meaningful and more difficult to update.  For example, what if we wanted to adjust the "medium" padding used around titles or buttons or something, but we didn't want to change the padding around empty text in lists?  If they are using the same variable, it would all have to change, because there is no contextual link between the variable and where it is being used.

Reply:
I don't feel super strongly about it FWIW, so long as we are consistent about spacing and basing it on the 4/8/12/16px rhythm 

I originally added this because I was thinking about SolveME / Metrodora - we slightly tweaked this and ended up using 24px padding instead of 16px in a lot of places to provide a little more breathing room and a simpler, less busy display.

It would perhaps be nice to achieve that kind of thing without CSS overrides aplenty like I ended up doing for Solve; however:
- Don't think it's a massive difference or super high priority necessarily
- I think it probably only applies for specific components/situations; lots of edge cases where it could break things otherwise

So probably the right solution here is to have a CSS variable that is used only in certain places to be able to "stretch" the UI a bit.  That being said again not really that concerned about it